starter-code/styles/base.css

```diff
@@ -11,6 +12,10 @@ hr {
   width: 80%;
 }
 
+img {
+  width: 100%;
```

Comment:
This should be `max-width` instead of just `width` so that the images do not expand to fill the entire page.